### PR TITLE
doc: update CHANGELOG

### PR DESCRIPTION
**Pull request by scotthart**
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/9606)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -116,16 +116,69 @@ may elicit a deprecation warning. See [#9086] for more details.
...
+
+* fix(spanner): avoid evaluation-order issue in function arguments ([#9452](https://github.com/googleapis/google-cloud-cpp/pull/9452))
+
+### [Storage](/google/cloud/storage/README.md)
+
+* feat(storage): experimental options to tune stall timeouts ([#9593](https://github.com/googleapis/google-cloud-cpp/pull/9593))
+* feat(storage): add debugging headers to `ObjectWriteStream` ([#9580](https://github.com/googleapis/google-cloud-cpp/pull/9580))
+* fix(storage): no char for `std::uniform_int_distribution` ([#9509](https://github.com/googleapis/google-cloud-cpp/pull/9509))
+* feat: improve error messages for access token errors ([#9485](https://github.com/googleapis/google-cloud-cpp/pull/9485))
+* feat(GCS+gRPC): more efficient `WriteObject()` stall timeouts ([#9576](https://github.com/googleapis/google-cloud-cpp/pull/9576))
```
Comment (coryan) [inline CHANGELOG.md:152]: We have not been documenting the changes to `GCS+gRPC`.  Please remove all of these.

File: CHANGELOG.md [hunk 2]
```diff
@@ -116,16 +116,69 @@ may elicit a deprecation warning. See [#9086] for more details.
...
+
+### [Assured Workloads](/google/cloud/assuredworkloads/README.md)
+
+* fix(assuredworkloads): reenable on windows ([#9467](https://github.com/googleapis/google-cloud-cpp/pull/9467))
+
+### [Common Libraries](/google/cloud/README.md)
+
+* fix(rest): build failure with MSVC+x86 ([#9586](https://github.com/googleapis/google-cloud-cpp/pull/9586))
+* feat(common): make the RPC log even more readable ([#9561](https://github.com/googleapis/google-cloud-cpp/pull/9561))
+* fix(asset): reenable on macOS ([#9468](https://github.com/googleapis/google-cloud-cpp/pull/9468))
```
Comment (dbolduc) [inline CHANGELOG.md:178]: I think this should be its own thing, like you did for Assured Workloads

File: CHANGELOG.md [hunk 3]
```diff
@@ -116,16 +116,69 @@ may elicit a deprecation warning. See [#9086] for more details.
...
+* feat(GCS+gRPC): implement `DeleteDefaultObjectAcl()` ([#9473](https://github.com/googleapis/google-cloud-cpp/pull/9473))
+* feat(GCS+gRPC): implement `CreateDefaultObjectAcl()` ([#9465](https://github.com/googleapis/google-cloud-cpp/pull/9465))
+* feat(GCS+gRPC): implement `GetDefaultObjectAcl()` ([#9458](https://github.com/googleapis/google-cloud-cpp/pull/9458))
+* feat(GCS+gRPC): implement `ListDefaultObjectAcl()` ([#9455](https://github.com/googleapis/google-cloud-cpp/pull/9455))
+
+### [Stackdriver Debugger](/google/cloud/debugger/README.md)
+
+* doc(debugger): announce deprecation ([#9552](https://github.com/googleapis/google-cloud-cpp/pull/9552))
+
+### [Assured Workloads](/google/cloud/assuredworkloads/README.md)
```
Comment (dbolduc) [inline CHANGELOG.md:170]: move ahead of `Bigtable` ?

File: CHANGELOG.md [hunk 4]
```diff
@@ -116,16 +116,69 @@ may elicit a deprecation warning. See [#9086] for more details.
...
+* feat(GCS+gRPC): implement `DeleteResumableUpload()` ([#9550](https://github.com/googleapis/google-cloud-cpp/pull/9550))
+* feat(GCS+gRPC): implement `PatchDefaultObjectAcl()` ([#9487](https://github.com/googleapis/google-cloud-cpp/pull/9487))
+* feat(GCS+gRPC): implement `UpdateDefaultObjectAcl()` ([#9476](https://github.com/googleapis/google-cloud-cpp/pull/9476))
+* feat(storage): support Bucket custom placement config ([#9481](https://github.com/googleapis/google-cloud-cpp/pull/9481))
+* feat(GCS+gRPC): use ETag proto fields ([#9472](https://github.com/googleapis/google-cloud-cpp/pull/9472))
+* feat(GCS+gRPC): implement `DeleteDefaultObjectAcl()` ([#9473](https://github.com/googleapis/google-cloud-cpp/pull/9473))
+* feat(GCS+gRPC): implement `CreateDefaultObjectAcl()` ([#9465](https://github.com/googleapis/google-cloud-cpp/pull/9465))
+* feat(GCS+gRPC): implement `GetDefaultObjectAcl()` ([#9458](https://github.com/googleapis/google-cloud-cpp/pull/9458))
+* feat(GCS+gRPC): implement `ListDefaultObjectAcl()` ([#9455](https://github.com/googleapis/google-cloud-cpp/pull/9455))
```
Comment (dbolduc) [inline CHANGELOG.md:164]: We have not been including the GCS+gRPC changes in the CHANGELOG

File: CHANGELOG.md [hunk 5]
```diff
@@ -116,16 +116,69 @@ may elicit a deprecation warning. See [#9086] for more details.
...
+
+* doc(debugger): announce deprecation ([#9552](https://github.com/googleapis/google-cloud-cpp/pull/9552))
+
+### [Assured Workloads](/google/cloud/assuredworkloads/README.md)
+
+* fix(assuredworkloads): reenable on windows ([#9467](https://github.com/googleapis/google-cloud-cpp/pull/9467))
+
+### [Common Libraries](/google/cloud/README.md)
+
+* fix(rest): build failure with MSVC+x86 ([#9586](https://github.com/googleapis/google-cloud-cpp/pull/9586))
```
Comment (dbolduc) [inline CHANGELOG.md:176]: `rest` is internal, so no need for a changelog entry

File: CHANGELOG.md [hunk 6]
```diff
@@ -116,16 +116,69 @@ may elicit a deprecation warning. See [#9086] for more details.
...
+* feat(common): make the RPC log even more readable ([#9561](https://github.com/googleapis/google-cloud-cpp/pull/9561))
+* fix(asset): reenable on macOS ([#9468](https://github.com/googleapis/google-cloud-cpp/pull/9468))
+* feat: improve error messages for access token errors ([#9485](https://github.com/googleapis/google-cloud-cpp/pull/9485))
+* feat(common): make the RPC log even more readable ([#9477](https://github.com/googleapis/google-cloud-cpp/pull/9477))
+* feat(rest): reduce contention in `PooledCurlHandleFactory` ([#9457](https://github.com/googleapis/google-cloud-cpp/pull/9457))
```
Comment (dbolduc) [inline CHANGELOG.md:181]: same

File: CHANGELOG.md [hunk 7]
```diff
@@ -116,16 +116,69 @@ may elicit a deprecation warning. See [#9086] for more details.
...
+* feat(GCS+gRPC): implement `PatchDefaultObjectAcl()` ([#9487](https://github.com/googleapis/google-cloud-cpp/pull/9487))
+* feat(GCS+gRPC): implement `UpdateDefaultObjectAcl()` ([#9476](https://github.com/googleapis/google-cloud-cpp/pull/9476))
+* feat(storage): support Bucket custom placement config ([#9481](https://github.com/googleapis/google-cloud-cpp/pull/9481))
+* feat(GCS+gRPC): use ETag proto fields ([#9472](https://github.com/googleapis/google-cloud-cpp/pull/9472))
+* feat(GCS+gRPC): implement `DeleteDefaultObjectAcl()` ([#9473](https://github.com/googleapis/google-cloud-cpp/pull/9473))
+* feat(GCS+gRPC): implement `CreateDefaultObjectAcl()` ([#9465](https://github.com/googleapis/google-cloud-cpp/pull/9465))
+* feat(GCS+gRPC): implement `GetDefaultObjectAcl()` ([#9458](https://github.com/googleapis/google-cloud-cpp/pull/9458))
+* feat(GCS+gRPC): implement `ListDefaultObjectAcl()` ([#9455](https://github.com/googleapis/google-cloud-cpp/pull/9455))
+
+### [Stackdriver Debugger](/google/cloud/debugger/README.md)
```
Comment (dbolduc) [inline CHANGELOG.md:166]: move ahead of `Storage` ?

File: CHANGELOG.md (hunk 2 above)
Comment (coryan) [inline CHANGELOG.md:178]: This looks like it should be in a section like `Assured Workloads`.

File: CHANGELOG.md [hunk 8]
```diff
@@ -116,16 +116,69 @@ may elicit a deprecation warning. See [#9086] for more details.
...
+### [Spanner](/google/cloud/spanner/README.md)
+
+* fix(spanner): avoid evaluation-order issue in function arguments ([#9452](https://github.com/googleapis/google-cloud-cpp/pull/9452))
+
+### [Storage](/google/cloud/storage/README.md)
+
+* feat(storage): experimental options to tune stall timeouts ([#9593](https://github.com/googleapis/google-cloud-cpp/pull/9593))
+* feat(storage): add debugging headers to `ObjectWriteStream` ([#9580](https://github.com/googleapis/google-cloud-cpp/pull/9580))
+* fix(storage): no char for `std::uniform_int_distribution` ([#9509](https://github.com/googleapis/google-cloud-cpp/pull/9509))
+* feat: improve error messages for access token errors ([#9485](https://github.com/googleapis/google-cloud-cpp/pull/9485))
```
Comment (dbolduc) [inline CHANGELOG.md:151]: this one also shows up in common. I think we can delete the storage entry

**Review by scotthart — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/9606)* status: 0 of 1 files reviewed, 8 unresolved discussions (waiting on @coryan and @dbolduc)

___
*[`CHANGELOG.md` line 151 at r1](https://reviewable.io/reviews/googleapis/google-cloud-cpp/9606#-N8Q0jkT2Eo-jFvGzNbY-r1-151:-N8Q8q203tkFCMlki0X5:b-896fix) ([raw file](https://github.com/googleapis/google-cloud-cpp/blob/121e8d866c48e812d376b3e07b8ba213a8500ab8/CHANGELOG.md#L151)):*
<details><summary><i>Previously, dbolduc (Darren Bolduc) wrote…</i></summary><blockquote>

this one also shows up in common. I think we can delete the storage entry
</blockquote></details>

Done.
___
*[`CHANGELOG.md` line 152 at r1](https://reviewable.io/reviews/googleapis/google-cloud-cpp/9606#-N8Q0jkT2Eo-jFvGzNbY-r1-152:-N8Q8qPt18tQEWVDFGUL:b-896fix) ([raw file](https://github.com/googleapis/google-cloud-cpp/blob/121e8d866c48e812d376b3e07b8ba213a8500ab8/CHANGELOG.md#L152)):*
<details><summary><i>Previously, coryan (Carlos O'Ryan) wrote…</i></summary><blockquote>

We have not been documenting the changes to `GCS+gRPC`.  Please remove all of these.
</blockquote></details>

Done.
___
*[`CHANGELOG.md` line 164 at r1](https://reviewable.io/reviews/googleapis/google-cloud-cpp/9606#-N8Q0jkT2Eo-jFvGzNbY-r1-164:-N8Q8qzi30TAnjJE72Tj:b-896fix) ([raw file](https://github.com/googleapis/google-cloud-cpp/blob/121e8d866c48e812d376b3e07b8ba213a8500ab8/CHANGELOG.md#L164)):*
<details><summary><i>Previously, dbolduc (Darren Bolduc) wrote…</i></summary><blockquote>

We have not been including the GCS+gRPC changes in the CHANGELOG
</blockquote></details>

Done.
___
*[`CHANGELOG.md` line 166 at r1](https://reviewable.io/reviews/googleapis/google-cloud-cpp/9606#-N8Q0jkT2Eo-jFvGzNbY-r1-166:-N8Q8rO-9IUKZ_dLt9UH:b-896fix) ([raw file](https://github.com/googleapis/google-cloud-cpp/blob/121e8d866c48e812d376b3e07b8ba213a8500ab8/CHANGELOG.md#L166)):*
<details><summary><i>Previously, dbolduc (Darren Bolduc) wrote…</i></summary><blockquote>

move ahead of `Storage` ?
</blockquote></details>

Done.
___
*[`CHANGELOG.md` line 170 at r1](https://reviewable.io/reviews/googleapis/google-cloud-cpp/9606#-N8Q0jkT2Eo-jFvGzNbY-r1-170:-N8Q8rui-nFjhKRAlgWv:b-896fix) ([raw file](https://github.com/googleapis/google-cloud-cpp/blob/121e8d866c48e812d376b3e07b8ba213a8500ab8/CHANGELOG.md#L170)):*
<details><summary><i>Previously, dbolduc (Darren Bolduc) wrote…</i></summary><blockquote>

move ahead of `Bigtable` ?
</blockquote></details>

Done.
___
*[`CHANGELOG.md` line 176 at r1](https://reviewable.io/reviews/googleapis/google-cloud-cpp/9606#-N8Q0jkT2Eo-jFvGzNbY-r1-176:-N8Q8sNv9sbDUdldwKIb:b-896fix) ([raw file](https://github.com/googleapis/google-cloud-cpp/blob/121e8d866c48e812d376b3e07b8ba213a8500ab8/CHANGELOG.md#L176)):*
<details><summary><i>Previously, dbolduc (Darren Bolduc) wrote…</i></summary><blockquote>

`rest` is internal, so no need for a changelog entry
</blockquote></details>

Done.
___
*[`CHANGELOG.md` line 178 at r1](https://reviewable.io/reviews/googleapis/google-cloud-cpp/9606#-N8Q0jkT2Eo-jFvGzNbY-r1-178:-N8Q8sy47NDaqlg7PL21:b-896fix) ([raw file](https://github.com/googleapis/google-cloud-cpp/blob/121e8d866c48e812d376b3e07b8ba213a8500ab8/CHANGELOG.md#L178)):*
<details><summary><i>Previously, coryan (Carlos O'Ryan) wrote…</i></summary><blockquote>

This looks like it should be in a section like `Assured Workloads`.
</blockquote></details>

Done.
___
*[`CHANGELOG.md` line 181 at r1](https://reviewable.io/reviews/googleapis/google-cloud-cpp/9606#-N8Q0jkT2Eo-jFvGzNbY-r1-181:-N8Q8toW2qI1F90pJzTO:b-896fix) ([raw file](https://github.com/googleapis/google-cloud-cpp/blob/121e8d866c48e812d376b3e07b8ba213a8500ab8/CHANGELOG.md#L181)):*
<details><summary><i>Previously, dbolduc (Darren Bolduc) wrote…</i></summary><blockquote>

same
</blockquote></details>

Done.

**Review by dbolduc — APPROVED** [review]

**Review by coryan — APPROVED** [review]

**Pull request merged**